framework/include/base/NonlinearSystem.h

```diff
@@ -515,6 +517,7 @@ class NonlinearSystem : public SystemTempl<TransientNonlinearImplicitSystem>,
   KernelWarehouse _kernels;
   MooseObjectWarehouse<ScalarKernel> _scalar_kernels;
   MooseObjectWarehouse<DGKernel> _dg_kernels;
+  MooseObjectWarehouse<InterfaceKernel> _interface_kernels;
```

Comment:
You made me very happy by using this without you needing to write a new warehouse class.